### DISCUSSION TITLE
Package it to docker

### DISCUSSION THREAD

**Discussion by vietdevne** [hacksider/Deep-Live-Cam#60] [Ideas]
I believe we should package this app into Docker so that it can run on different operating systems. We have installed it on several different machines and encountered many errors during both the build and run phases of the app.
What do you think about that?

**Comment by hacksider** [comment]
Actually, you can create a docker for yourself and I'll link it for everyone. I have uploaded CUDA packge few days ago though (for nvidia users) in this [link](https://github.com/hacksider/Deep-Live-Cam/releases/tag/cuda_deep-live-cam)

**Reply by vietdevne** [reply]
> Thực ra, bạn có thể tự tạo một docker và tôi sẽ liên kết nó cho mọi người. Tôi đã tải gói CUDA lên cách đây vài ngày (dành cho người dùng nvidia) trong [liên kết này](https://github.com/hacksider/Deep-Live-Cam/releases/tag/cuda_deep-live-cam)

it too awesome!
